Release notes 7.1 (updates)

fpetru opened the pull request:
It contains a major update to the Release notes for 7.1. It fully replaces PR #449 .
Associated with the Github issue #423 .

Here is a quick preview:

![image](https://user-images.githubusercontent.com/5551796/115988325-75deca80-a5b9-11eb-8448-c1c75e547117.png)

MichaIng commented on docs/releases.md:
```diff
@@ -4,23 +4,71 @@
+Many thanks to all contributors and beta testers :octicons-heart-16: :octicons-heart-16: !! With your help we could achieve all of these updates ! Install or upgrade to this latest update - it has more things we hope you would enjoy ! Share your feedback on [Twitter](https://twitter.com/DietPi_), [DietPi Forum](https://dietpi.com/phpbb/viewforum.php?f=9&sid=e6fcb5de3776c2d8eb1b22cc1fb24ca5), and  ...
 
-### Merged software list
+As a new thing, we already scheduled the next update. DietPi 7.2 will be launched on Friday, 27 May !
```
It will be Saturday 29 May 😃. Saturday is a good day for me to do last reviews and tests.

fpetru replied:
:) very sorry, I'm changing it now !!

MichaIng commented on docs/releases.md:
```diff
@@ -44,36 +92,24 @@ _Note:_ Since multiple software titles were listening on the ports `5000` and `8
     !!! hint ""
 
-        - It has now support for resizing F2FS and Btrfs filesystems as well as format- and filesystem check & repair support for XFS filesystems.
+        - The script now supports resizing F2FS and Btrfs filesystems as well as format, filesystem check and repair support for XFS filesystems.
```
> format- and filesystem check & repair support for XFS

I chose that, meaning "format support for XFS" and "filesystem check & repair support for XFS". The ampersand used here as it is used within the scripts menu as well "Check & Repair". Probably we should rework it more to clear that all last things are meant for XFS? Like

```
and the XFS filesystem type is now fully supported to format drives with and have them checked for filesystem errors.
```

fpetru replied:
@MichaIng - Here it's just an idea:

_The script supports resizing for F2FS and Btrfs filesystems. The XFS filesystem type is fully supported to format drives with and have them checked for filesystem errors._

Please adjust it as needed. Thanks

MichaIng replied:
LGTM. Keep the "now" inside to indicate that it's a change/addition?

Joulinar commented on docs/releases.md:
```diff
@@ -4,23 +4,71 @@
+Many thanks to all contributors and beta testers :octicons-heart-16: :octicons-heart-16: !! With your help we could achieve all of these updates ! Install or upgrade to this latest update - it has more things we hope you would enjoy ! Share your feedback on [Twitter](https://twitter.com/DietPi_), [DietPi Forum](https://dietpi.com/phpbb/viewforum.php?f=9&sid=e6fcb5de3776c2d8eb1b22cc1fb24ca5), and  ...
 
-### Merged software list
+As a new thing, we already scheduled the next update. DietPi 7.2 will be launched on 29 May !
```
Should we really do this exact statement? I mean what happen if we have a delay? We could keep this date internal, to avoid to create expectation we could not fulfil. There is a reason why RPi devs never populate a release date. It's done when it is done 😉

MichaIng replied:
My idea indeed was to make it fix from now on, so that we and users can plan with it. Aiming a date/range internally and not fulfilling it, was what we/me did mostly until now 😅, but I think it has quite a value to have reliable release schedules. Fixing the date more publicly helps to assure that it will be kept. First beta will start exactly one ween before, on Mai 22, so all larger changes that have been finished until then, are part of the next release, everything else will go into the next. That makes more sense than delaying all changes by a month to get the second half into the same release 😄.

Even if there is something urgent not making it until the date, I see no issue in pushing a small hotfix release a week later or so. The smaller the releases, the less changes/issues are accumulated, hence the smoother updates theoretically will be. E.g. a major part of all bug reports we receive is due to accumulated APT upgrades being all done at once together with the DietPi update.

Joulinar replied:
I agree, quite a large amount of issues is related to `apt` failure. But this will not go away with shorter release cycle. We already inform user about possible `apt` updates. As well, not every user will login that often. Means they will have the same amount of `apt` updates still before execution of DietPi update. We have users not login for ages or even running Stretch. Based on statistics, more than 20% running v6.33 still. 😉

MichaIng replied:
A shorter release schedules still implies the chance that users update earlier, of course it does not guarantee it. More regular APT upgrades can prevent failures with it, e.g. when multiple upgrades, especially also in combination with 3rd party repos, collide in a way. Also the chance that unavoidable APT errors are split means a chance that individual errors can be debugged easier compared to multiple individual errors that appear together or even affect each other are coming together.

The daily APT check indeed helps to prevent accumulated updates, but some may disable it while having DietPi update notification enabled. Jep it does not guarantee anything for individual cases but the chances for more regular updates done by users are undoubtedly better when updates are available more regular, which is enough for me to accept it as pro argument 😄. The other thing is broken or weakened install options or features due to upstream/package changes and allowing us and users to better plan implementation/system update schedules.

Joulinar replied:
Don't get me wrong. I don't argue against it. Just tried to look from all angles into it.

Joulinar commented on docs/releases.md:
```diff
@@ -85,9 +121,14 @@ _Note:_ Since multiple software titles were listening on the ports `5000` and `8
 
+### Changes / Improvements / Optimisations in 7.1
+
+- **`DietPi-Arr_to_RAM`** :octicons-arrow-right-16: Support for [Sonarr](../software/bittorrent/#sonarr) v3 and [Radarr](../software/bittorrent/#radarr) v3 has been added. On first link to RAM, a script `/mnt/dietpi_userdata/(sonarr|radarr|lidarr)/dietpi-arr_to_RAM.sh` is created, which allows updating the linked database backups via the programs "Custom Script" feature. Since v3 it is no longer  ...
```
we would need to remove the ` around DietPi-Arr_to_RAM

MichaIng replied:
Agreed. I'll create a docs section for it by times, then we can make it a link. It requires adding "Arr" to the wordlist, to we can start using pirate speak, arrrr 😄.

MichaIng commented:
I added all image width + height + lazy loading, extended alt tags and removed some doubled spaces and "to" copy&paste mistakes in the older logs.

fpetru commented:
> I added all image width + height + lazy loading, extended alt tags and removed some doubled spaces and "to" copy & paste mistakes in the older logs.

Thank you ! I missed these updates.
